lighthouse-plugin-ad-speed-insights/index.js

```diff
@@ -20,7 +20,7 @@ const yargs = require('yargs');
 async function main() {
   if (!yargs.argv.configPath) {
     // Avoid duplicating the option
-    const configPath = require.resolve('./config');
+    const configPath = require.resolve('../config');
```

Comment:
seems like `index.js` should still be outside the plugin directory (alongside the config file)? Since it's going to run lighthouse with the plugin, it's not part of the plugin

Reply:
Done.